config/etc/system-image/writable-paths

```diff
@@ -94,7 +78,9 @@
 /etc/systemd/system.conf.d              auto                    persistent  transition  none
 /etc/systemd/user.conf.d                auto                    persistent  transition  none
 /etc/systemd/logind.conf.d              auto                    persistent  transition  none
+# FIXME: what is this used for?
 /etc/iproute2                           auto                    persistent  transition  none
```

Comment:
this is used for adding more complex routing tables than just a default route ... it was added in the course to fix https://bugs.launchpad.net/snappy/+bug/1658298 and is also part of the network-control interface